app/validators/has_one_title_validator.rb

```diff
@@ -0,0 +1,8 @@
+# validates that the title has at least one title
+class HasOneTitleValidator < ActiveModel::Validator
```

Comment:
Isn't it at `app/has_one_title_validator.rb` too then? There's a renamed without any changes part of this commit.